Sources/Helper/Theme.swift

```diff
@@ -72,6 +72,8 @@ public struct Theme {
     public struct Text {
         /// The text primary color
         public static var color = UIColor.black
+        /// The text disabled color
+        public static var disabledColor = UIColor(rgb: 0x8e8e93)
```

Comment:
There shouldn't be disabled color. It should be same as label color

Reply:
fixed